simpleviewinc/tap-events-force
# Zen 495 no sessions msg

**Ticket**: [ZEN-495](https://jira.simpleviewtools.com/browse/zen-495)


## Context

* When there are no sessions to display for a day, we should show a message explaining that

## Goal

* Implement the component and it's styling

## Updates

* `src/components/grid/emptyDayMessage.js`
  * the component for the message
* `src/components/sessions.js`
  * updated the `AgendaSessions` component to use `EmptyDayMessage`
* `src/theme/components/grid/emptyDayMessage.js`
  * styles for the component

## Testing

* `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:zen-495-no-sessions-msg`
* navigate to http://evf-zen-495-no-sessions-msg.local.kegdev.xyz/
* Open Test Data JSON editor
* select `no data - no sessions` option
   * [image](https://user-images.githubusercontent.com/3317835/102820186-11ed9080-4392-11eb-940a-31b219a841b7.png)
* Verify you see the message `There are no sessions to display for this day`
* Refresh the page
* open the filter modal and filter on "Waiting List"
* verify the same message appears on Day 1 and Day 2
